utils/src/main/java/uk/gov/di/authentication/utils/lambda/BulkUserEmailSenderScheduledEventHandler.java

```diff
@@ -90,7 +90,7 @@ public Void handleRequest(ScheduledEvent event, Context context) {
         final long bulkUserEmailBatchPauseDuration =
                 configurationService.getBulkUserEmailBatchPauseDuration();
         final List<String> bulkUserEmailExcludedTermsAndConditions =
-                configurationService.getBulkUserEmailExcludedTermsAndConditions();
+                configurationService.getBulkUserEmailIncludedTermsAndConditions();
```

Comment:
I think we need to change more than this here, since the logic in the handler is still excluding these (e.g. the logic in `sendEmailIfRequiredAndUpdateStatus` will need changing, and some tests will need updating too

Reply:
thanks updated